sys/syslog/backend_file.c

```diff
@@ -0,0 +1,211 @@
+
+#include <fcntl.h>
+#include <errno.h>
+#include <string.h>
+
+#include "syslog.h"
+#include "backend.h"
+#include "vfs.h"
+#include "mutex.h"
+
+#define ENABLE_DEBUG    (1)
```

Comment:
```suggestion
#define ENABLE_DEBUG    0
```

Please don't use braces here, as this prevents using it with the `IS_ACTIVE()` helper macro.